tests/unittests/functional/test_entity.py

```diff
@@ -22,7 +22,7 @@ class TestEntity(unittest.TestCase):
     def test_init(self):
         d = {'a': 'A', 'b': 'B', 'c': 'C'}
         e1 = Entity()
-        e2 = Entity(d)
+        e2 = Entity(**d)
```

Comment:
Entity class support loading data from a dict directly.

Reply:
I'm facing a problem: the argument name `target` may conflict with the keywords argument when converting a pandas data frame to entities.